domain/src/test/java/org/oppia/android/domain/platformparameter/PlatformParameterModuleTest.kt

```diff
@@ -6,109 +6,152 @@ import androidx.test.ext.junit.runners.AndroidJUnit4
     )
   }
 
+  private val partialPlatformParameterMapWithValues by lazy {
+    val mockIntegerPlatformParameter = PlatformParameter.newBuilder()
+      .setInteger(TEST_INTEGER_PARAM_VALUE).build()
+    val mockBooleanPlatformParameter = PlatformParameter.newBuilder()
+      .setBoolean(TEST_BOOLEAN_PARAM_VALUE).build()
+
+    mapOf<String, PlatformParameter>(
+      TEST_INTEGER_PARAM_NAME to mockIntegerPlatformParameter,
+      TEST_BOOLEAN_PARAM_NAME to mockBooleanPlatformParameter
+    )
+  }
+
   @Test
   fun testModule_initPlatformParameterMap_retrieveTestStringParameter_returnsParamValue() {
-    setUpTestApplicationComponent(mockPlatformParameterMap)
-    assertThat(testStringPlatformParameter.value).isEqualTo(TEST_STRING_PARAM_VALUE)
+    setUpTestApplicationComponent(platformParameterMapWithValues)
+    assertThat(testStringPlatformParameter.get().value).isEqualTo(TEST_STRING_PARAM_VALUE)
   }
 
   @Test
   fun testModule_doNotInitPlatformParameterMap_retrieveTestStringParameter_returnsDefaultValue() {
     setUpTestApplicationComponent(mapOf())
-    assertThat(testStringPlatformParameter.value).isEqualTo(TEST_STRING_PARAM_DEFAULT_VALUE)
+    assertThat(testStringPlatformParameter.get().value).isEqualTo(TEST_STRING_PARAM_DEFAULT_VALUE)
   }
 
   @Test
   fun testModule_initPlatformParameterMap_retrieveTestIntegerParameter_returnsParamValue() {
-    setUpTestApplicationComponent(mockPlatformParameterMap)
-    assertThat(testIntegerPlatformParameter.value).isEqualTo(TEST_INTEGER_PARAM_VALUE)
+    setUpTestApplicationComponent(platformParameterMapWithValues)
+    assertThat(testIntegerPlatformParameter.get().value).isEqualTo(TEST_INTEGER_PARAM_VALUE)
   }
 
   @Test
   fun testModule_doNotInitPlatformParameterMap_retrieveTestIntegerParameter_returnsDefaultValue() {
     setUpTestApplicationComponent(mapOf())
-    assertThat(testIntegerPlatformParameter.value).isEqualTo(TEST_INTEGER_PARAM_DEFAULT_VALUE)
+    assertThat(testIntegerPlatformParameter.get().value).isEqualTo(TEST_INTEGER_PARAM_DEFAULT_VALUE)
   }
 
   @Test
   fun testModule_initPlatformParameterMap_retrieveTestBooleanParameter_returnsParamValue() {
-    setUpTestApplicationComponent(mockPlatformParameterMap)
-    assertThat(testBooleanPlatformParameter.value).isEqualTo(TEST_BOOLEAN_PARAM_VALUE)
+    setUpTestApplicationComponent(platformParameterMapWithValues)
+    assertThat(testBooleanPlatformParameter.get().value).isEqualTo(TEST_BOOLEAN_PARAM_VALUE)
   }
 
   @Test
   fun testModule_doNotInitPlatformParameterMap_retrieveTestBooleanParameter_returnsDefaultValue() {
     setUpTestApplicationComponent(mapOf())
-    assertThat(testBooleanPlatformParameter.value).isEqualTo(TEST_BOOLEAN_PARAM_DEFAULT_VALUE)
+    assertThat(testBooleanPlatformParameter.get().value).isEqualTo(TEST_BOOLEAN_PARAM_DEFAULT_VALUE)
+  }
+
+  @Test
+  fun testModule_initPlatformParameterMapPartially_retrieveTestStringParameter_returnsDefaultValue() { // ktlint-disable max-line-length
```

Comment:
maybe `testModule_initPlatformParameterMapPartially_returnsExpectedDefaultsAndValues` since this checks all the types.

Reply:
Let's try to avoid using `// ktlint-disable` max-line-length and we can try to keep things under 100 char long.

Reply:
+1

Reply:
Done